src/metabase/query_processor/store.clj

```diff
@@ -111,6 +111,7 @@
   (s/both
    (mi/InstanceOf Field)
    {:name                               su/NonBlankString
+    :table_id                           su/IntGreaterThanZero
```

Comment:
not strictly related but when writing a test I noticed it wasn't erroring if I forgot to include this